packages/@cdk-cosmos/core/src/helpers/cdk-pipeline.ts

```diff
@@ -88,6 +88,10 @@ export class CdkPipeline extends Construct {
       environment: {
         buildImage: LinuxBuildImage.STANDARD_3_0,
         environmentVariables: {
+          NPM_KEY: {
```

Comment:
I believe you have tested and verified this in the container!

Reply:
not yet ...

Reply:
verified locally and works fine..!!!